sub-packages/bionemo-evo2/tests/bionemo/evo2/run/test_train.py

```diff
@@ -42,11 +42,12 @@ def test_train_evo2_runs(tmp_path, num_steps=5):
     # a local copy of the environment
     env = dict(**os.environ)
     env["MASTER_PORT"] = str(open_port)
+    experiment_name = "test"
 
     # Build the command string.
     # Note: The command assumes that `train_evo2` is in your PATH.
     command = (
-        f"train_evo2 --mock-data --experiment-dir {tmp_path}/test_train "
+        f"train_evo2 --mock-data --result-dir {tmp_path} --experiment-name {experiment_name}"
         "--model-size 1b_nv --num-layers 4 --hybrid-override-pattern SDH* "
         "--no-activation-checkpointing --add-bias-output "
         f"--max-steps {num_steps} --warmup-steps 1 --no-wandb "
```

Comment:
you should remove the `--no-wandb` option from this and we should remove it from our docs too. This is now controlled by not setting `--wandb-project`

Reply:
thanks John, removed